.env.example

```diff
@@ -28,9 +28,9 @@ HOST_WORKDIR=`pwd`
 # ensure that this path exists at the host!
 HOST_CACHEDIR=/netscratch/$USER/.cache_slurm
 # change this if you have installed miniconda to another location
-HOST_CONDA_ENVS_DIR=/netscratch/$USER/miniconda3/envs
+HOST_CONDA_ENVS_DIR=/netscratch/$USER/miniconda3/
 # change this if you use another Enroot image with a different conda location
-CONTAINER_CONDA_ENVS_DIR=/opt/conda/envs
+CONTAINER_CONDA_ENVS_DIR=/opt/conda/
```

Comment:
Why are there slashes at the end of the paths?

Reply:
functionally there is no difference between having the slash in the end vs not having the slash in the end. I believe having the slash in the end is the general UNIX convention, but i can remove those if you want me to?

Reply:
hmm not so sure about that, I found just that, for instance: https://en.wikipedia.org/wiki/Talk:Path_(computing)#Filepath_Misconceptions

Not 100% sure, but I think it is a bit better to remove them.

Reply:
got it